src/buyingcatalogue/Chart.yaml

```diff
@@ -92,6 +92,6 @@ dependencies:
   version: ~0.1.0
 - name: pb-ac-tests
   condition: pb-ac-tests.enabled
-  version: ~0.5.0-alpha
+  version: 0.5.0-alpha.159
```

Comment:
Should we be updating this to an alpha sub release in Master?

Reply:
Ideally it'd be a real version, but one isn't ready yet.